### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by ChenMiya** [josephpmcdonald/tensorflowtrials#3]
Why the accuracy is so high ？I have the same question with hudong.
Test Accuracy is about 0.9881 when the epoch is 75.
Looking forward to your reply.

**Comment by jpmcd** [comment]
Here is the output I get for the last two epochs and the test accuracy. Note the validation and test accuracy are both around 80%. The training accuracy is high but that's because the size of the dataset is relatively small.

```
Epoch: 74 Learning rate: 0.032
batch size 20
Testing 1998 samples:
batch 98 / 99
Training Accuracy = 1.0000, time = 9.128 seconds

batch size 20
Testing 105 samples:
batch 4 / 5
Valid Accuracy = 0.7600

Epoch: 75 Learning rate: 0.031
batch size 20
Testing 1998 samples:
batch 98 / 99
Training Accuracy = 0.9995, time = 9.164 seconds

batch size 20
Testing 105 samples:
batch 4 / 5
Valid Accuracy = 0.7600

batch size 20
Testing 2254 samples:
batch 111 / 112
Test Accuracy = 0.7987
```

**Comment by ChenMiya** [comment]
Thank you for your reply. And Can you give me the code and the dataset you ran? My email adress is 1905318074@qq.com,sincerely looking forward to your reply.

**Comment by ChenMiya** [comment]
Thank you for your reply. And Can you give me the code and the dataset you ran? My email address is 1905318074@qq.com,sincerely looking forward to your reply.

**Comment by jpmcd** [comment]
Sorry, I forgot I had commented out the automatic download for the dataset. The data set is here:
http://www.iro.umontreal.ca/~lisa/deep/data/imdb.pkl
You may need to adjust some of the path strings for your system.

**Comment by jpmcd** [comment]
closing this
